### DISCUSSION TITLE
BlurryImage it's done!!!

### DISCUSSION THREAD

**Discussion by PieroCastillo** [AvaloniaUI/Avalonia#5708] [General]
Hi, I finished another control for my controls lib, coming soon will be available in nuget! (or use it cloning the repo) 
This control blur a image, works equals to Image, but accepts IBitmap only, another image type don't.

https://user-images.githubusercontent.com/39932324/112191282-db632400-8bd3-11eb-9677-a39a35d8d0df.mp4

(sorry for the background music)

**Comment by nielsbosma** [comment]
@PieroCastillo did you ever release this? I'm looking for a way to do a panel that blur's it's content.

**Reply by Mrxx99** [reply]
It is in his library Aura.UI
https://github.com/PieroCastillo/Aura.UI

**Reply by nielsbosma** [reply]
Thanks. Do you know if there's any way to blur all the contents of a panel?

**Reply by timunie** [reply]
You got answers in your other thread you opened already.
